tests/to_figure_test.py

```diff
@@ -0,0 +1,10 @@
+from ladybug_charts.to_figure import heatmap
+from ladybug_charts.to_df import create_df, DataPoint
+from plotly.graph_objects import Figure
+
+
+def test_heatmap():
+    epw_path = r'tests/assets/weather/boston.epw'
+    df = create_df(epw_path)[0]
```

Comment:
Really nice to see how easy the process is. For the sake of clarity for DF as dataframe vs DF as Dragonfly vs DF as Daylight Factor, I suggest changing the name to `create_data_frame` or even better - inject it to EPW class and make it `to_dataframe`.

```suggestion
    df = create_dataframe(epw_path)[0]
```

Preferred approach in my mind:

```python
epw = EPW(epw_path )
epw_df = epw.to_dataframe()

## or even

epw = EPW(epw_path )
figure = epw.relative_humidity.heatmap()

```